src/tools/qt-gui/org.libelektra.elektra-qt-editor.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Elektra Qt Editor
+GenericName=Qt Configuration Editor for Elektra
```

Comment:
Yes, according to [Arch Linux Wiki](https://wiki.archlinux.org/index.php/Desktop_entries) it should be "what you would generally call an application that does what this specific application offers", so rather "Elektra Configuration Editor" or even "Configuration Editor".